frontend/src/metabase/containers/UndoListing.css

```diff
@@ -9,7 +9,6 @@
   composes: bordered rounded shadowed from "style";
   composes: relative from "style";
   composes: flex align-center from "style";
-  width: 320px;
```

Comment:
Where do undos get their size now?

Reply:
It should just be based on their content now. Figured that's the better gauge / place to trim things rather than setting the same width globally.

Reply:
That might look weird if we have multiple toasts on screen with different sizes, but that's a bit of an edge case. Maybe a `min-width` at least?